### PR TITLE
Temporary fix for debian7

### PR DESCRIPTION
**Pull request by ph**

### CODE REVIEW

**Comment by ph** [comment]
Need to be backported to 7.x 7.0 6.7

File: metricbeat/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 FROM golang:1.10.3
 
 RUN \
-    apt-get update \
+    apt-get update -o Acquire::Check-Valid-Until=false \
```
Comment (andrewkroh) [inline metricbeat/Dockerfile]: These ones probably aren't necessary since this is based on debian9.

File: x-pack/auditbeat/magefile.go [hunk 2]
```diff
@@ -253,7 +253,7 @@ func installDependencies(pkg, arch string) error {
 		}
 	}
 
-	if err := sh.Run("apt-get", "update"); err != nil {
+	if err := sh.Run("apt-get", "update", "-o", "Acquire::Check-Valid-Until=false"); err != nil {
```
Comment (andrewkroh) [inline x-pack/auditbeat/magefile.go]: Let's put a "TODO" in here as well saying that this is only for debian 7 and should be removed when move to a newer OS. This flag is going to be used unnecessary when building using non-debian7 images (like when making the linux/arm binaries) and we should remove it soonish.

**Comment by ph** [comment]
jenkins test this

**Comment by andrewkroh** [comment]
jenkins, package this

**Pull request merged**